### ISSUE TITLE
ColorKeyword erroneously finds map-get arguments

### ISSUE THREAD

**Issue by jamesarosen** [brigade/scss-lint#821]
If I have a map like
```scss
$my-colors: (
  red: #de2424,
  green: #28d424,
  blue: #2820e0
);
```
and I use it like
```scss
.some-thing {
  border-color: map-get($my-colors, green);
}
```
then the ColorKeyword checker alerts me that `green` should be written in hexadecimal form. But it isn't a color keyword here; it's a table key.

Unfortunately, I'm not sure there's a way to distinguish between that and `darken(green, 10%)`, which should be caught by ColorKeyword.

**Comment by sds** [comment]
Hey @jamesarosen,

From the perspective of the SCSS parser (which is outside of our control), `green` _is_ a color keyword &mdash;that's what it's parsed as and represented as in the parse tree (`Sass::Script::Value::Color`).

You can avoid this by wrapping it in quotes so it is parsed as a string, e.g. `'green'`. There's not much SCSS-Lint can do here since it can't determine that you actually "meant" for the color keyword to be "just a string."

Hope that helps!

**Comment by jamesarosen** [comment]
Sadly, `map-get($some-map, foo)` and `map-get($some-map, 'foo')` are not the same :/

**Comment by sds** [comment]
Hmmm, it's unfortunate that the SCSS parser differentiates the two, though in hindsight it makes sense. I've added an exception to allow color keywords as arguments in `map-*` function calls in de91368.
